### ISSUE TITLE
Build failure with aeson 0.10

### ISSUE THREAD

**Issue by bergmark** [srijs/haskell-ndjson-conduit#1]
```
Building ndjson-conduit-0.1.0.2...
Preprocessing library ndjson-conduit-0.1.0.2...
[1 of 1] Compiling Data.Conduit.JSON.NewlineDelimited ( src/Data/Conduit/JSON/NewlineDelimited.hs, dist/build/Data/Conduit/JSON/NewlineDelimited.o )

src/Data/Conduit/JSON/NewlineDelimited.hs:45:19:
    Not in scope: ‘A.encodeToByteStringBuilder’
```

Also see http://matrix.hackage.haskell.org/package/ndjson-conduit#GHC-7.10/ndjson-conduit-0.1.0.2

Stackage just upgraded to aeson-0.10 which means I need to disable ndjson-conduit for now. Please send a pull request to re-enable it when this is fixed. If this happens before new years ndjson-conduit will still be included in LTS 4.

**Comment by srijs** [comment]
Thanks, I'll see what I can do.

**Comment by srijs** [comment]
@bergmark Published a new version to cabal (0.1.0.3) that should fix the issue.

**Comment by bergmark** [comment]
Thanks!
